py/get_tiles.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python
```

Comment:
im just wondering if we really need the whole class here? i mean dont we just need the bounding box logic? the other logic is useful but not in this context right?